moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/bullet_utils.h

```diff
@@ -119,10 +119,10 @@ class CollisionObjectWrapper : public btCollisionObject
                          const std::set<std::string>& touch_links);
 
   /** \brief Bitfield specifies to which group the object belongs */
-  short int m_collision_filter_group;
+  short int m_collisionFilterGroup;
```

Comment:
if this doesn't have to conform to bullet standards, it should be `collisionFilterGroup_`.

Reply:
I think it makes sense to keep it in bullet naming convention as this class derives from [btCollisionObject](https://pybullet.org/Bullet/BulletFull/classbtCollisionObject.html) and it actually mimics the behavior of the [broadphase proxy collision object](https://pybullet.org/Bullet/BulletFull/structbtBroadphaseProxy.html) which has the same member variables.

Reply:
Ah, I see, didn't know it wasn't bullet format before.

Reply:
Our clang-tidy cleanups will likely change this in the future... I'm not sure its worth keeping in bullet format